github/AuthenticatedUser.py

```diff
@@ -1213,6 +1214,17 @@ def get_migrations(self):
             }
         )
 
+    def get_organization_membership(self, org):
+        """
+        :calls: `GET /user/memberships/orgs/:org <https://developer.github.com/v3/orgs/members/#get-your-organization-membership>`_
+        :rtype: :class:`github.Membership.Membership`
+        """
+
+        assert isinstance(org, int)
+
+        headers, data = self._requester.requestJsonAndCheck("GET", "/user/memberships/orgs/" + str(org))
```

Comment:
Would you mind following the same whitespace rules for this call that everywhere uses?

Reply:
Done @s-t-e-v-e-n-k . New check has been triggered.

Reply:
@s-t-e-v-e-n-k Please check now

Reply:
That looks unchanged. From further up in the file:
```
        headers, data = self._requester.requestJsonAndCheck(
            "POST",
            "/user/migrations",
            input=post_parameters,
            headers={
                "Accept": Consts.mediaTypeMigrationPreview
            }
        )
```
It does not need to be quite that complex, but I'd like it to use the same indent.

Reply:
These lines are not part of my changes. Should I make the change for all similar instances?

Reply:
I'm sorry, to be clear, I want you to change your addition to match that indenting. Like so:
```
        headers, data = self._requester.requestJsonAndCheck(
            "GET",
            "/user/memberships/orgs/" + str(org)
        )
```